src/components/ThemedVideo/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint jsx-a11y/media-has-caption: 0 */
+import React, { useEffect, useState } from "react";
+import { useTheme } from "../../context/ThemeContext";
+import { passImage } from "../../utils/helpers";
+
+interface ThemedVideoProps {
+  width?: string;
+  muted?: boolean;
+  controls?: boolean;
+  loop?: boolean;
+  title?: string;
+  src: Array<string> | string;
+  type?: any;
+}
+
+const ThemedVideo: React.FC<ThemedVideoProps> = ({
+  width = "100%",
+  muted,
+  controls,
+  loop,
+  title = "",
+  src = "",
+  type = "video/mp4",
+}) => {
+  const { theme } = useTheme();
+  const vidSrc = passImage(src, theme);
+  const [currentSrc, setSrc] = useState(vidSrc);
```

Comment:
```suggestion
  const [currentSrc, setCurrentSrc] = useState(vidSrc);
```
think react convention is to use:  variableName, setVariableName